cl/lib/elasticsearch_utils.py

```diff
@@ -673,3 +674,30 @@ def fetch_es_results(
         if settings.DEBUG is True:
             traceback.print_exc()
     return [], 0, error
+
+
+def do_es_podcast_query(
+    search_query: Search,
+    cd: CleanData,
+    override_params: dict[str, str] | None = None,
+    rows: int = 20,
+) -> list[Document]:
+    """Execute an Elasticsearch query for podcasts.
+
+    :param search_query: Elasticsearch DSL Search object
+    :param cd: The query CleanedData
+    :param override_params: Search parameters to override, if any
+    :param rows: Number of rows (items) to be retrieved in the response
+    :return: A list of documents from the Elasticsearch response.
+    """
+
+    if override_params:
+        cd.update(override_params)
+    s, total_query_results, top_hits_limit = build_es_main_query(
+        search_query, cd
+    )
+    response = s.extra(from_=0, size=rows).execute()
+    items = []
+    for item in response:
+        items.append(item)
+    return items
```

Comment:
Huh. Is this the same as:

```suggestion
    return [item for item in response]
```

If so, I guess we're doing this because `response` is some iterable object type and if we don't do it here, the feed API is sad later?

Reply:
Well now we're returning the `response` directly here. I remember experiencing some issues when returning the direct response, but that might have been due to a malformed query. I've confirmed that the `Response` object has a `__iter__` method, which returns the documents contained in the hits key, so it's okay to return it directly.